package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "geostyler-mapbox-parser",
+  "version": "0.0.1",
+  "description": "GeoStyler-Style-Parser implementation for Mapbox",
+  "main": "index.js",
```

Comment:
Main should point to the build index.js version
Please also add `"types": "[BUILDPATH]/index.d.ts"`